#request
---
#response mensaje
string object_name
bool success
string status_message